### ISSUE TITLE
Error with Imgix transformer when passing an AssetTransform object

### ISSUE THREAD

**Issue by benface** [aelvan/Imager-Craft#237]
### Repro steps

1. Use the following Imager settings:
```
    'transformer' => 'imgix',
    'useForNativeTransforms' => true,
    'imgixProfile' => 'default',
    'imgixApiKey' => getenv('IMGIX_API_KEY'),
    'imgixConfig' => [
        'default' => [
            'domains' => ['yourdomain.imgix.net'],
            'signKey' => getenv('IMGIX_SIGN_KEY'),
        ],
    ],
```
2. Create a `test` transform in the CP, with any settings.
3. Run this template (you gotta have at least one asset in the DB):
```
{{ craft.app.assets.getAssetUrl(craft.assets.one(), craft.app.assetTransforms.getTransformByHandle('test')) }}
```

Expected result:  No error.
Actual result:
```
yii\base\ErrorException: rawurlencode() expects parameter 1 to be string, array given in /Users/benface/Desktop/Freelance/project/vendor/imgix/imgix-php/src/Imgix/UrlHelper.php:74
```

**Comment by aelvan** [comment]
This was fixed in v2.1.10, thanks for reporting!

**Comment by benface** [comment]
Wow! That was quick. Thank you very much!
